jvb/src/main/java/org/jitsi/videobridge/rest/root/colibri/v2/conferences/Conferences.java

```diff
@@ -0,0 +1,162 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.rest.root.colibri.v2.conferences;
+
+import org.jitsi.videobridge.*;
+import org.jitsi.videobridge.rest.*;
+import org.jitsi.videobridge.rest.annotations.*;
+import org.jitsi.videobridge.rest.exceptions.*;
+import org.jitsi.xmpp.extensions.colibri2.*;
+import org.jitsi.xmpp.extensions.colibri2.json.*;
+import org.jivesoftware.smack.packet.*;
+import org.json.simple.*;
+import org.json.simple.parser.*;
+
+import javax.inject.*;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
+import java.util.*;
+
+@Path("/colibri/v2/conferences")
+@EnabledByConfig(RestApis.COLIBRI)
+public class Conferences
+{
+    @Inject
+    private Videobridge videobridge;
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public String getConferences()
+    {
+        /* TODO */
+        throw new ServerErrorException(Response.Status.NOT_IMPLEMENTED);
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/{meetingId}")
+    public String getConference(@PathParam("meetingId") String meetingId)
+    {
+        Conference conference = videobridge.getConferenceByMeetingId(meetingId);
+
+        if (conference == null)
+        {
+            throw new NotFoundException();
+        }
+
+        /* TODO */
+        throw new ServerErrorException(Response.Status.NOT_IMPLEMENTED);
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/{meetingId}/dominant-speaker-identification")
+    public String getDominantSpeakerIdentification(@PathParam("meetingId") String meetingId)
+    {
+        Conference conference = videobridge.getConferenceByMeetingId(meetingId);
+
+        if (conference == null)
+        {
+            throw new NotFoundException();
+        }
+
+        ConferenceSpeechActivity conferenceSpeechActivity = conference.getSpeechActivity();
+
+        return conferenceSpeechActivity.getDebugState().toJSONString();
+    }
+
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public String createConference(String requestBody)
+    {
+        Object requestJson;
+        try
+        {
+            requestJson = new JSONParser().parse(requestBody);
+            if (!(requestJson instanceof JSONObject))
+            {
+                throw new BadRequestException();
+            }
+        }
+        catch (ParseException pe)
+        {
+            throw new BadRequestExceptionWithMessage(
+                    "Failed to create conference, could not parse JSON: " + pe.getMessage());
+        }
+
+        ConferenceModifyIQ conferenceModifyIQ
```

Comment:
Maybe try/catch any deserialization exceptions? They should be equivalent to bad JSON